content/docs/guides/contribute/review-process.en.md

```diff
@@ -0,0 +1,20 @@
+Here's a suggested workflow. The quotation marks indicate the **necessity** of communication to ensure the smooth flow of PR validation:
+
+```mermaid
+sequenceDiagram
+  actor A as PR author
+  actor R as Reviewer/Maintainer
+  
+  A->>R: ASKS "I need a review"
+  R->>A: ANSWERS ✅ "Ok" or ❌ "not ok"
+  
+  R-->>+A: COMMENTS, asks for changments
```

Comment:
```suggestion
```

Duplicate of "approves or requests changes

Reply:
In reality, lines 8 and 9 correspond to the setting up of the agreement between the PR reviewer and producer.